CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/AnalyticViewTopComponent.java

```diff
@@ -217,97 +203,14 @@ protected void componentShowing() {
-                                final boolean transactionSelected = readableGraph.getBooleanValue(transactionSelectedAttribute, transactionId);
-                                if (transactionSelected) {
-                                    selected.add(transactionId);
-                                }
-                            }
-                            break;
-                        default:
-                            break;
-                    }
-                } finally {
-                    readableGraph.release();
-                }
-
-                result.setSelectionOnVisualisation(elementType, selected);
-            }
-        }
-    }
-
-    @PluginInfo(pluginType = PluginType.SELECTION, tags = {PluginTags.SELECT})
-    public static class SelectOnGraphPlugin extends SimpleEditPlugin {
-
-        final GraphElementType elementType;
-        final List<Integer> elementIds;
-
-        public SelectOnGraphPlugin(final GraphElementType elementType, final List<Integer> elementIds) {
-            this.elementType = elementType;
-            this.elementIds = elementIds;
-        }
-
-        @Override
-        public String getName() {
-            return AnalyticViewTopComponent.AnalyticController.SELECT_ON_GRAPH_PLUGIN_NAME;
-        }
-
-        @Override
-        protected void edit(final GraphWriteMethods graph, final PluginInteraction interaction, final PluginParameters parameters) throws InterruptedException, PluginException {
-            switch (elementType) {
-                case VERTEX:
-                    final int vertexSelectedAttribute = VisualConcept.VertexAttribute.SELECTED.get(graph);
-                    final int vertexCount = graph.getVertexCount();
-                    for (int vertexPosition = 0; vertexPosition < vertexCount; vertexPosition++) {
-                        final int vertexId = graph.getVertex(vertexPosition);
-                        graph.setBooleanValue(vertexSelectedAttribute, vertexId, elementIds.contains(vertexId));
-                    }
-                    break;
-                case TRANSACTION:
-                    final int transactionSelectedAttribute = VisualConcept.TransactionAttribute.SELECTED.get(graph);
-                    final int transactionCount = graph.getTransactionCount();
-                    for (int transactionPosition = 0; transactionPosition < transactionCount; transactionPosition++) {
-                        final int transactionId = graph.getTransaction(transactionPosition);
-                        graph.setBooleanValue(transactionSelectedAttribute, transactionId, elementIds.contains(transactionId));
-                    }
-                    break;
-                default:
-                    break;
-            }
-        }
+    
+    @Override
+    protected void handleComponentClosed() {
+        super.handleComponentClosed();
+        analyticController.deactivateResultUpdates();
```

Comment:
This appears to only remove changes on the active graph. It will need to go through all open graphs.